docs/user-guide/src/SUMMARY.md

```diff
@@ -4,17 +4,19 @@
 - [Installation](install.md)
 - [Basic Usage](basic.md)
 - [Guided Tour](tour/summary.md)
+  - [Setup](tour/setup.md)
   - [Getting Started](tour/getting-started.md)
   - [New](tour/new.md)
   - [Push](tour/push.md)
   - [Pop](tour/pop.md)
   - [Testing](tour/testing.md)
-  - [A Bad Stack](tour/bad-stack.md)
-  - [Options](tour/options.md)
+  - [Option](tour/option.md)
   - [Generics](tour/generics.md)
   - [Peek](tour/peek.md)
+  - [Pledges (mutable peek)](tour/pledges.md)
   - [Final Code](tour/final.md)
-  - [Pledges](tour/pledges.md)
+  - [Loop Invariants](tour/loop_invariants.md)
+  - [Counterexample](tour/counterexamples.md)
 - [Verification Features](verify/summary.md)
```

Comment:
This should also include `prusti-feature.md` (and maybe other changes to the ToC I missed?)